jupyterbook/content/ioos_installation_conda.md

```diff
@@ -118,6 +118,10 @@ Once the environment is done building, you can activate it by typing:
 conda activate IOOS
 ```
 
+Now you can start hacking the notebook with `jupyter nbclassic`,
+for the old style notebooks,
+ or `jupyter-lab` for the new IDE interface.
```

Comment:
This is not available in our env:
```
(base) C:\>conda activate ioos
(ioos) C:\>jupyter-lab
'jupyter-lab' is not recognized as an internal or external command,
operable program or batch file.
```

Reply:
It is added in this PR.

Reply:
duh - sorry for that oversight

Reply:
To be fair I also thought it was already there.